content/iOS/Practices/iOS-practices.md

```diff
@@ -0,0 +1,118 @@
+# A few iOS practices that can go a long way when maintaining your projects.
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team who writes about the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
```

Comment:
using my favorite feature: 
![image](https://user-images.githubusercontent.com/8755889/83076698-63c47700-a04c-11ea-9c34-382a043d54e8.png)

I entered because I thought it was weird that it was a generic tech-blog link and not something more personal like the linkedin link.

Reply:
https://www.rootstrap.com/blog/. this is RS blog url, with blog instead of tech-blog

Reply:
I wanted to reference @glm4 iOS base article but apparently I screw up the copy pasta 🍝

Reply:
mmmm I think here you want to mention your blogs profile, since it is inside a Germán Stábile text. You actually mention later in the article an iOS base article and that link is ok.